proof_of_stake/src/lib.rs

```diff
@@ -2355,13 +2355,13 @@ pub fn change_validator_commission_rate<S>(
 where
     S: StorageRead + StorageWrite,
 {
-    // if new_rate < Uint::zero() {
-    //     return Err(CommissionRateChangeError::NegativeRate(
-    //         new_rate,
-    //         validator.clone(),
-    //     )
-    //     .into());
-    // }
+    if new_rate.is_negative() {
```

Comment:
we should also check that it's not greater than 1 here

Reply:
done now

Reply:
thx!